test/cli/rbs-cache-dir/test.out

```diff
@@ -16,10 +17,6 @@ end
 No errors! Great job.
 class <emptyTree><<C <root>>> < (::<todo sym>)
   class <emptyTree>::<C A><<C <todo sym>>> < (::<todo sym>)
-    ::Sorbet::Private::Static.sig(::T::Sig::WithoutRuntime) do ||
-      <self>.returns(<emptyTree>::<C Integer>)
-    end
-
```

Comment:
Note that on the second run, where the `--enable-experimental-rbs-signatures` flag is no longer being passed, that we're no longer seeing this show up in the parse output.